CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/http/apis/OpenAi.cs

```diff
@@ -256,14 +256,23 @@ public static class ChatGptExtensions {
             self.Add(new ChatGpt.Line(ChatGpt.Role.system, content: CreateJsonInstructions(exampleResponse)));
         }
 
-        public static string CreateJsonInstructions<T>(T exampleResponse) {
+        public static string CreateJsonInstructions<T>(params T[] exampleResponses) {
+            if (exampleResponses.IsNullOrEmpty()) return null;
```

Comment:
This path here should fail fast and throw an exception instead of returning null